test/e2e/framework/config.go

```diff
@@ -75,3 +77,11 @@ func ParseConfig() (*Config, error) {
 	}
 	return c, nil
 }
+
+func (c *Config) GetOsSpecificVersionedFilePath(baseFileName string, versionIndex int32) string {
+	if c.IsWindowsTest {
+		return fmt.Sprintf("%s\\%s", baseFileName, versionIndex)
+	}
+
+	return fmt.Sprintf("%s/%s", baseFileName, versionIndex)
```

Comment:
that was how I originally had it; however, the windows e2e tests ended up failing because the file path was incorrect. My thinking was that test code is running on a linux machine (even when it's operating against a windows aks agent pool) and so I updated it to flag based on if it's testing windows or not